packages/web/src/services/audius-backend/Tipping.ts

```diff
@@ -58,3 +61,71 @@ export const fetchSupporters = async ({
     return []
   }
 }
+
+export type UserTipRequest = {
+  userId?: string
+  limit?: number
+  offset?: number
+  receiverMinFollowers?: number
+  receiverIsVerified?: boolean
+  currentUserFollows?: 'sender' | 'receiver' | 'sender_or_receiver'
+  uniqueBy?: 'sender' | 'receiver'
+  minSlot?: number
+  maxSlot?: number
+  txSignatures?: string[]
+}
+type UserTipOmitIds = 'sender_id' | 'receiver_id' | 'followee_supporter_ids'
+type UserTipResponse = Omit<UserTip, UserTipOmitIds> & {
+  sender: any
```

Comment:
just wondering why cant be string or smthg?

Reply:
because it's a user-like object